docs/JAMULUS_PROTOCOL.md

```diff
@@ -0,0 +1,181 @@
+
+The Jamulus Audio Protocol
+==
```

Comment:
Just a small comment on the style: Could you please follow the style like on the website (Headings are denoted with `#` not `=`).

Reply:
It also may be worthy to make a reference to the wireshark dissector at https://github.com/softins/jamulus-wireshark as whoever reads this doc, most surely be interested in additional troubleshooting tools.

Reply:
It's mentioned in line 8?

Reply:
> Just a small comment on the style: Could you please follow the style like on the website (Headings are denoted with `#` not `=`).

Done.